erc20-load-test-tool/src/main/java/com/vmware/ethereum/service/SecureTokenFactory.java

```diff
@@ -84,6 +92,16 @@ public SecurityToken getSecureToken() {
         .send();
   }
 
+  /** Returns gas price */
+  private BigInteger getGasPrice() {
```

Comment:
Maybe we can fail the call instead of sneakily setting gas price to 0. So I suggest you remove this method and just add `web3j.ethGasPrice().send().getGasPrice()` where it is being invoked.

Reply:
done